### DISCUSSION TITLE
Member list opens each "check_interval" seconds

### DISCUSSION THREAD

**Discussion by farnuca** [TobiasPankner/Teams-Auto-Joiner#155] [Q&A] [answered]
This is not a bug, I just have a "problem": i need the program to stop looking for new meetings when there's one running, so i set "pause_search": true... But each "check_interval" seconds, the meeting shows the member list for a few seconds. Is there any way to avoid this?

**Comment by TobiasPankner** [comment, marked as answer]
No, currently there is no way to avoid this. Sorry
